lib/psabpf_pipeline.c

```diff
@@ -389,13 +389,13 @@ int psabpf_pipeline_add_port(psabpf_context_t *ctx, const char *interface)
     build_ebpf_prog_filename(pinned_file, sizeof(pinned_file), ctx, XDP_HELPER_PROG);
     isXDP = access(pinned_file, F_OK) != 0;
 
-    int ifindex = (int) if_nametoindex(interface);
-    if (!ifindex) {
+    *ifindex = (int) if_nametoindex(interface);
```

Comment:
I think this argument should be optional, I mean `if (ifindex != NULL) *ifindex = ...;`

Reply:
`if_nametoindex` returns 0 if tere is no such interface. 
https://man7.org/linux/man-pages/man3/if_nametoindex.3.html

Reply:
I mean that pointer to result should be checked for NULL-ness. `if_nametoindex ` have nothing to the address of that variable.